packages/stylist/src/lib.rs

```diff
@@ -118,11 +69,35 @@
 //! There's theming example using
 //! [Yew Context API](https://github.com/futursolo/stylist-rs/tree/master/examples/yew-theme-context).
 //!
+//! ### Style API
+//!
+//! If you want to parse a string into a style at runtime, you need to enable the
+//! `parser` feature. You can then use [`Style::new`], passing a `str`, `String`
+//! or `Cow<'a, str>`:
+//!
+//! ```rust
+//! use stylist::Style;
+//!
+//! let style = Style::new(
+//!     r#"
+//!         background-color: red;
+//!
+//!         .nested {
+//!             background-color: blue;
+//!             width: 100px
+//!         }
+//!     "#,
+//! )
+//! .expect("Failed to create style");
+//! ```
+//!
 //! ## Features Flags
 //!
 //! - `macros`: Enabled by default, this flag enables procedural macro support.
 //! - `random`: Enabled by default, this flag uses `fastrand` crate to generate a random class name.
 //!   Disabling this flag will opt for a class name that is counter-based.
+//! - `parser`: Disabled by default, this flag enables runtime parsing of styles from strings. You
```

Comment:
parser is actually enabled by default.

I am indifferent in disabling or enabling this by default.
I felt disabling this might be better but we do lose debug assertions on interpolation values. We can make the parser by default debug only, but there doesn't seem to be a way to exclude dependencies on release anyways so the benefit seems not big enough.

Reply:
Hm you are right. Can we maybe internally consume `stylist-core/parser` (trusting the tree shaking and dead code elimination in release mode) but still only publicly expose the additional features with `stylist/parser` turned on?

Reply:
Split the feature into `parser` for the public part and `debug_parser` for the debug assertions.